app/middlewares/Dev/IpRestrictMiddleware.php

```diff
@@ -46,13 +46,15 @@ public function __construct(HttpKernelInterface $app)
      *
      * {@inheritdoc}
      */
-    public function handle(Request $request, $type = self::MASTER_REQUEST, $catch = true)
+    /**
+     * This check prevents access to debug front controllers
+     * that are deployed by accident to production servers.
+     *
+     * {@inheritdoc}
+     */
+    public function handle(Request $request, $type = self::MAIN_REQUEST, $catch = true)
     {
-        if (in_array($request->getClientIp(), $this->allowedIps) || false !== getenv('DDEV_TLD')) {
-            return $this->app->handle($request, $type, $catch);
-        }
-
-        return new Response('You are not allowed to access this file.', 403);
```

Comment:
The tests are failing because you removed this. What was the reason for removing this? Please update just the constant.

Reply:
Oh, I'm sorry about that. I made those changes on my local machine when I was trying to access the development environment. I will rectify it soon and create a PR. 

Thanks!

Reply:
No problem. No need to create a new PR. Just push a new commit to this one. Check the Github Actions for any other issues it reports. It should be all green for reviews to be requested.

Reply:
Done!